connect_labs/workflow/tests/test_kmc_programme_metrics_action.py

```
@@ -0,0 +1,107 @@
+    has to reach the UI, or a green verdict on their photos reads as settled."""
+    assert {"GHI", "Kikapu"} <= UNVERIFIED_SCALE_LLOS
+    assert "scale_unverified_llos" in DEFINITION["config"]
+
+
+def test_an_llo_running_both_hardware_types_is_marked_unverified():
+    """Routing by LLO is only sound while an LLO's hardware is consistent. If one
+    ever runs both, picking either reader silently mis-reads half its photos - so
+    the collapse records the conflict instead of resolving it."""
+    from connect_labs.workflow.templates.kmc_programme_metrics import _scale_agent_by_llo
+
+    real_by_llo, real_conflicts = _scale_agent_by_llo()
```

Ruff (0.16.3)

[warning] Unpacked variable `real_by_llo` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)